websites/H/HegartyMaths/presence.ts

```diff
@@ -0,0 +1,141 @@
+const presence = new Presence({
+		clientId: "935600572457963561"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+function getQueryVariable(v: string) {
+	const vars = window.location.search.substring(1).split("&");
+	for (let i = 0; i < vars.length; i++)
+		if (vars[i].split("=")[0] === v) return vars[i].split("=")[1];
+	return false;
+}
```

Comment:
Use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams).

Reply:
Changed in b9d2ed4